libs/execution_context/ebpf_program.c

```diff
@@ -2242,7 +2242,7 @@ _ebpf_program_test_run_work_item(_In_ cxplat_preemptible_work_item_t* work_item,
                 ebpf_lower_irql(old_irql);
 
                 // Reacquire the CPU.
-                old_irql = ebpf_raise_irql(context->required_irql);
+                old_irql = ebpf_raise_irql(context->maximum_irql);
```

Comment:
nit: should we do this only when the current IRQL < `context->maximum_irql`

Reply:
So don't change IRQL if maximum IRQL == passive?

Reply:
Yes. Not a major issue though. Will probably save some cycles calling `KeRaiseIrql` / `KeLowerIrql`.

Reply:
It's outside of the block where we are measuring time, so it should not impact the performance numbers gathered.